web/src/components/Skeleton/SurveyListSkeleton.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import styled, { DefaultTheme } from 'styled-components';
+
+const ListTable = styled.table`
+  display: flex;
+  flex-direction: column;
+  padding: 3vh 5vw 1vh 5vw;
+  background-color: ${(props) => props.theme.colors.container};
+`;
+
+const ListHead = styled.thead``;
+
+const ListBody = styled.tbody``;
+
+const ListRow = styled.tr`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+const Item = styled.td`
+  height: 22px;
+  margin: 2px;
+  padding: 2vh;
+  font-size: 1.7vh;
+  font-weight: bold;
+  border-radius: 5px;
+  background-color: ${(props) => props.theme.colors.header};
+`;
+
+const HeadItem = styled.th`
+  margin: 2px;
+  padding: 2vh;
+  font-size: 2vh;
+  font-weight: bold;
+  text-align: center;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const Title = styled(Item)`
+  min-width: 15vh;
+  flex: 1;
+`;
+
+const Authlist = styled(Item)`
+  min-width: 160px;
+  width: 20vw;
+`;
+
+const EndDate = styled(Item)`
+  min-width: 150px;
+  width: 13vw;
+  text-align: center;
+
+  @media screen and (max-width: 900px) {
+    display: none;
+    }
+  } 
+`;
+
+const HeadTitle = styled(HeadItem)`
+  min-width: 15vh;
+  flex: 1;
+`;
+
+const HeadAuthList = styled(HeadItem)`
+  min-width: 160px;
+  width: 20vw;
+`;
+
+const HeadEndDate = styled(HeadItem)`
+  min-width: 150px;
+  width: 13vw;
+  
+  @media screen and (max-width: 900px) {
+    display: none;
+    }
+  } 
+`;
+
+interface SkeletonProps {
+  theme: DefaultTheme;
+}
+export default function SurveyListSkeleton({ theme }: SkeletonProps) {
+  const surveyIndex = [1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
Well, as we're fetching 8 rows for our API for now but this could be changed in a variety of reasons. So please avoid hard-coded one though we're currently fetching 8 data for each page. This kind of code makes it really hard to debug.

And also to create an array of numbers, please check https://stackoverflow.com/a/65244361/17320013 for performance.